include/veriblock/consts.hpp

```diff
@@ -12,6 +14,7 @@ constexpr const auto MAX_HEADER_SIZE_PUBLICATION_DATA = 1024;
 constexpr const auto MAX_PAYOUT_SIZE_PUBLICATION_DATA = 100;
 constexpr const auto MAX_CONTEXT_SIZE_PUBLICATION_DATA = 100;
 constexpr const auto SHA256_HASH_SIZE = 32;
+constexpr const auto VBLAKE_BLOCK_SIZE = 24;
```

Comment:
it is bad name for a const, because actually VBK_BLOCK_SIZE is 64 bytes, but here you mean HASH of block. 
Better name it `VBLAKE_BLOCK_HASH_SIZE`